File: cpp/home_work_1/task1.cpp
#include <iostream>

int main() {
    int x,y,z, max;
    std::cout << "Please Enter X, Y, Z" << std::endl;
    std::cin >> x >> y >> z;
    
    max = x;
    
    if (y > max) {
        max = y;
    }
    
    if (z > max) {
        max = z;
    }

    std::cout << "Max number is " << max << std::endl;

    return 0;
}

File: java/home_work_1/task1.java
import java.util.*;
class Task1 {
    public static void main(String[] args) {
        
            int x,y,z, max;
            System.out.println("Please Enter X, Y, Z" + "\n");
            Scanner sx = new Scanner(System.in);
            Scanner sy = new Scanner(System.in);
            Scanner sz = new Scanner(System.in);
            x = sx.nextInt();
            y = sy.nextInt();
            z = sz.nextInt();
            
            max = x;
    
            if (y > max) {
                max = y;
            }
    
            if (z > max) {
                max = z;
            }

    System.out.println("Max number is " + max) ;
    }
}
